pkg/innerring/processors/governance/events.go

```diff
@@ -9,3 +17,25 @@ func (s Sync) MorphEvent() {}
 func NewSyncEvent() Sync {
 	return Sync{}
 }
+
+// Designate represents designation event of the mainnet RoleManagement contract.
+type Designate struct {
```

Comment:
I would recommend to move designate event to `morph/event/rolemanagement` package. Events in processor sub packages are tend to be internal, and designate event produced by external native contract.